kernel-default.properties

```diff
@@ -42,18 +42,25 @@ mosip.kernel.sms.sender=sender
 mosip.kernel.sms.unicode=unicode
 
 ## Email notification
-mosip.kernel.notification.email.from=mosipuser@gmail.com
-spring.mail.host=${email.smtp.host}
-spring.mail.username=${email.smtp.username}
-spring.mail.password=${email.smtp.secret}
-spring.mail.port=587
+mosip.kernel.notification.email.from={cipher}36c31e62f69626c1a027747fae8657588c6ab527322d6fbdf4ea5b1934732505
```

Comment:
these smtp related configuration change ar not required.

Reply:
smtp changes also required kernel notification service getting issue 
mail health check failed
health.89 : Mail health check failed
javax.mail.MessagingException: STARTTLS is required but host does not support STARTTLS
"statusCode":503